Writerside/topics/EventSource.md

```diff
@@ -0,0 +1,84 @@
+# EventSource
+
+An event source provides a translation layer between your <tooltip term="game server">game server</tooltip>, the <tooltip term="backend platform">backend platform.</tooltip> and <tooltip term="fleet manager">fleet manager</tooltip>.
+This way your game only needs to interface with the sidecar, and it can be deployed anywhere.
+
+Currently, Catena supports the following event sources:
+
+<snippet id="event-source-list">
+<ul>
+<li>Catena Backend Event Source - An event source which matches the <a href="Server-Manager.md">Catena Server Manager</a> API.</li>
+<li>Virtual Server Event Source - A mock server for simulating calls or doing zero-code implementations</li>
```

Comment:
Likely worth elaborating here as to why this is helpful. We understand it but I can imagine someone reading this for the first time will not.